### DISCUSSION TITLE
Finding where a color comes from

### DISCUSSION THREAD

**Discussion by elusian** [tsujan/Kvantum#971] [Q&A]
I'm trying to modify one of the themes (KvAdaptaDark), but I'm having issues with the color of some specific elements.
I modified both the kvconfig and the svg, and indeed most colors are ok, except (I will talk about the kvantummanager preview for simplicity):
- the tooltip text color (hovering the buttons from the top toolbar), which is always dark on dark background
- the background in text input in the top toolbar, which has the same color no matter the theme
- the "Button Box" label in the bottom of the first tab, which is always white-ish

In fact, apparently this is an issue that I have on all dark themes, even the provided ones.

I tried the brute force method of turning elements red in the theme to make them stand out, but even with everything red I could not find a setting that controlled those elements.

Is there a better way to figure out what needs to be changed in the theme? Some way to inspect the application and finding where the style comes from?

**Comment by tsujan** [comment]
Everything is explained in the files "Theme-Config" and "Theme-Making" in the "doc" folder inside the source.

> Is there a better way to figure out what needs to be changed in the theme?

If you mean better than trial and error, yes. The more you know about Qt widgets, the easier you could find them in the documentation.
